broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -139,6 +138,12 @@ public static BrokerController createBrokerController(String[] args) {
                 System.exit(-2);
             }
 
+            if (!nettyServerConfig.getBindIP().equals("0.0.0.0") &&
+                !nettyServerConfig.getBindIP().equals(brokerConfig.getBrokerIP1())) {
```

Comment:
The host to bind may be different with wildcard address and broker IP1. Broker IP1 is for clients to connect to brokers. For example, in the cloud environment, brokerIP1 is configured to its public IP. 
The ECS itself may have multiple network interfaces wiring virtually to different networks.

Reply:
In the cloud environment, if public IP is assigned to one virtual network interface, just set brokerIP1 and bindIP to public IP. More generally case is cloud host use nat, which means you can not bind to public IP directly because of OS limition.

<img width="641" alt="image" src="https://user-images.githubusercontent.com/32436597/188814482-1b505bc1-a701-474c-85de-aa9c7d9c135d.png">

In this case, you can set brokerIP1 to public IP and set bindIP to 0.0.0.0. The same is true for binding to multiple network interfaces.